### PR TITLE
Cleanup GetStartupInfo code

### PR DESCRIPTION
**Pull request by hughbe**
## Proposed Changes
- Cleanup GetStartupInfo code

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2805)

### CODE REVIEW

File: src/System.Windows.Forms.Primitives/src/Interop/Kernel32/Interop.STARTUPINFOW.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            public uint dwX;
+            public uint dwY;
+            public uint dwXSize;
+            public uint dwYSize;
+            public uint dwXCountChars;
+            public uint dwYCountChars;
+            public uint dwFillAttribute;
+            public STARTF dwFlags;
+            public ushort wShowWindow;
+            public ushort cbReserved2;
```
Comment (JuditRose) [inline src/System.Windows.Forms.Primitives/src/Interop/Kernel32/Interop.STARTUPINFOW.cs]: Why changing the type here?

Comment (RussKie) [reply src/System.Windows.Forms.Primitives/src/Interop/Kernel32/Interop.STARTUPINFOW.cs]: Please refer to https://github.com/dotnet/winforms/pull/2807#discussion_r375008348

File: src/System.Windows.Forms/src/System/Windows/Forms/Form.cs [hunk 2]
```diff
@@ -6118,20 +6118,20 @@ private void WmExitSizeMove(ref Message m)
...
                 {
-                    case (short)User32.SW.MAXIMIZE:
+                    case (ushort)User32.SW.MAXIMIZE:
                         WindowState = FormWindowState.Maximized;
                         break;
-                    case (short)User32.SW.MINIMIZE:
+                    case (ushort)User32.SW.MINIMIZE:
                         WindowState = FormWindowState.Minimized;
                         break;
                 }
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/Form.cs]: Can we instead update the `switch` to the following?
```cs
                switch ((User32.SW)si.wShowWindow)
                {
                    case User32.SW.MAXIMIZE:
                        WindowState = FormWindowState.Maximized;
                        break;
                    case User32.SW.MINIMIZE:
                        WindowState = FormWindowState.Minimized;
                        break;
                }
```

Comment (hughbe) [reply src/System.Windows.Forms/src/System/Windows/Forms/Form.cs]: sure! done

**Pull request merged**